src/screens/TripDetails/DepartureDetails/index.tsx

```diff
@@ -149,7 +150,7 @@ export default function DepartureDetails({navigation, route}: Props) {
             </View>
           )}
 
-          {!canSellTicketsForDeparture && (
+          {enable_ticketing && !canSellTicketsForDeparture && (
```

Comment:
Here I typically would extract it as a variable. For easier reading but also to name concepts to clearer convey intention.

Reply:
Might be a good plan yes. I have now introduced the isTicketingEnabledAndWeCantSellTicketForDeparture variable.